<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1392</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <widget class="QFrame" name="TopLevelFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="ControlFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="fileInfoLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Gotham</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:  grey;</string>
           </property>
           <property name="text">
            <string>File Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="SummaryFrame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="tempButtonLayerFrame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_13">
                 <property name="minimumSize">
                  <size>
                   <width>350</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame_11">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="stopButton">
                       <property name="minimumSize">
                        <size>
                         <width>150</width>
                         <height>150</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <family>Gotham</family>
                         <pointsize>16</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
    border: 1px solid rgb(87, 87, 87);

	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.188, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:50px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:focus {
	outline: none;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../../ControlCenter/src/ui/resources/resource.qrc">
                         <normaloff>:/Icons/img/icons/video-player-stop-button.png</normaloff>:/Icons/img/icons/video-player-stop-button.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>50</width>
                         <height>50</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="playPauseButton">
                       <property name="minimumSize">
                        <size>
                         <width>150</width>
                         <height>150</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <family>Gotham</family>
                         <pointsize>16</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
    border: 1px solid rgb(87, 87, 87);
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.188, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:50px;
    
}

QPushButton:pressed {

    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:focus {
	outline: none;
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../../ControlCenter/src/ui/resources/resource.qrc">
                         <normaloff>:/Icons/img/icons/play-button.png</normaloff>
                         <normalon>:/Icons/img/icons/pause-button.png</normalon>:/Icons/img/icons/play-button.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>50</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                       <property name="checked">
                        <bool>false</bool>
                       </property>
                       <property name="autoDefault">
                        <bool>false</bool>
                       </property>
                       <property name="default">
                        <bool>false</bool>
                       </property>
                       <property name="flat">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="fileFrame">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFrame" name="frame">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <item>
                      <widget class="QPushButton" name="loadFileButton">
                       <property name="minimumSize">
                        <size>
                         <width>150</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>200</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <family>Gotham</family>
                         <pointsize>16</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
    border: 1px solid rgb(87, 87, 87);

	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.188, stop:0 rgba(180, 180, 180, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:20px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}

QPushButton:focus {
	outline: none;
}</string>
                       </property>
                       <property name="text">
                        <string>Load Folder</string>
                       </property>
                       <property name="icon">
                        <iconset resource="../../../../ControlCenter/src/ui/resources/resource.qrc">
                         <normaloff>:/Icons/img/icons/folder.png</normaloff>:/Icons/img/icons/folder.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>20</width>
                         <height>20</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="layerPreviewFrame">
                    <property name="maximumSize">
                     <size>
                      <width>400</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_12">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QWidget" name="previewWidget" native="true">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>300</width>
                         <height>300</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>300</width>
                         <height>300</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background-color: rgb(232, 232, 232);
</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="fileInfoLabel_5">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>20</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <family>Gotham</family>
                         <pointsize>16</pointsize>
                         <weight>50</weight>
                         <bold>false</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">color:  grey;</string>
                       </property>
                       <property name="text">
                        <string>3D Preview</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="printProgressBar">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Gotham</family>
                <pointsize>16</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QProgressBar::chunk {

	background-color: qlineargradient(spread:pad, x1:0, y1:0.523, x2:0, y2:0.534, stop:0 rgba(130, 203, 117, 255), stop:1 rgba(66, 191, 85, 255));
border: 1px solid green;
}

QProgressBar {
    border: 1px solid rgb(87, 87, 87);
 /*  border-bottom-left-radius: 10px;
 border-bottom-right-radius: 10px;*/
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(150, 150, 150, 255), stop:1 rgba(180, 180, 180, 255));
}
</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="textVisible">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="format">
               <string>%p%</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="CameraFeedFrame">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>20000</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="layerPreviewWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>350</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(232, 232, 232);
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_12">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="fileInfoLabel_12">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Gotham</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:  grey;</string>
              </property>
              <property name="text">
               <string>Layer Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="subLayerPreviewWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>350</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(232, 232, 232);
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fileInfoLabel_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Gotham</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:  grey;</string>
           </property>
           <property name="text">
            <string>Sub-Layer Preview</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../ControlCenter/src/ui/resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
